packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -320,6 +322,15 @@ angular
             ),
           );
         }
+
+        return promise;
+      };
+
+      $scope.goToDeployWebSite = (domain) => {
+        return $state.go('app.hosting.dashboard.multisite.git-deployment', {
+          serviceName: $scope.hosting.serviceName,
+          path: domain.path,
```

Comment:
**niptick** : `$state.go(...)` should be defined in the routing file

Reply:
It was not easy because we don't have component file but with $scope.$resolve we made it. It's fixed.